README.md

```diff
@@ -63,6 +63,45 @@ Requests to cross-scheme subresources will no longer send `SameSite=Lax` or `Sam
 For example, if http://site.example embeds an image from https://site.example, `SameSite=Lax` and `SameSite=Strict` cookies will no longer be sent on the request to https://site.example, and any such cookies in the response will be ignored. 
 
 ## Questions
+### How does this affect iframes?
+#### The src attribute
+A request to get the content specified by the `src` attribute is treated as any other subresource request and is affected the same ways.
+Once the content is loaded it runs as cross-site or same-site depending on if the `src` value was cross-scheme or not, respectively.
+
+For example, the following request for a page will not be sent `SameSite=Lax` nor `SameSite=Strict` cookies (nor will its response be able to set those cookies) as it's cross-scheme. After the page is loaded nothing on the page is able to access those `SameSite` cookies because it's being run as cross-site.
```

Comment:
Nit: comma after loaded.

"For example, the following request for a page" and "After the page is loaded nothing on the page is able to access" reads to me like you're talking about the parent document rather than the iframe. Maybe you want to specify the iframe?

Although this entire example is a little odd since the iframe would be blocked as mixed content anyway. Flip the two around perhaps?

Reply:
Ah, good catch with the mixed content.
I'll rework the phrasing here.